### PR TITLE
Changes the client's requests from POST to GET.

### PR DESCRIPTION
**Pull request by hananothman**
This includes changing the URL request string to include the
request's parameters instead of posting the entire request proto.

### CODE REVIEW

File: api.go [hunk 1]
```diff
@@ -68,24 +74,15 @@ func newNetAPI(root string, key string, proxy string) (*netAPI, error) {
...
-	u := *a.url // Make a copy of URL
-	u.Path = requestPath
-	httpReq, err := http.NewRequest("POST", u.String(), bytes.NewReader(p))
-	httpReq.Header.Add("Content-Type", "application/x-protobuf")
+// doRequests performs a GET to requestPath. It automatically unmarshals the
+// response body payload as resp.
+func (a *netAPI) doRequest(ctx context.Context, urlString string, resp proto.Message) error {
+	httpReq, err := http.NewRequest("GET", urlString, nil)
+	httpReq.Header.Add("Content-Type", "application/json")
 	httpReq.Header.Add("User-Agent", "Webrisk-Client/0.1.2")
```
Comment (JJLionHeart) [inline api.go]: Please update the version. I think 0.1.3 should be fine.

File: api.go [hunk 2]
```diff
@@ -100,17 +97,38 @@ func (a *netAPI) doRequest(ctx context.Context, requestPath string, req proto.Me
 	if err != nil {
 		return err
 	}
-	return proto.Unmarshal(body, resp)
+	return jsonpb.UnmarshalString(string(body), resp)
```
Comment (JJLionHeart) [inline api.go]: Would this throw an exception or similar if the input is malformed?

File: webrisk_client.go [hunk 3]
```diff
@@ -447,7 +447,7 @@ func (sb *WebriskClient) LookupURLsContext(ctx context.Context, urls []string) (
 
 	for _, req := range reqs {
 		// Actually query the Web Risk API for exact full hash matches.
-		resp, err := sb.api.HashLookup(ctx, req)
+		resp, err := sb.api.HashLookup(ctx, req.HashPrefix, req.ThreatTypes)
```
Comment (JJLionHeart) [inline webrisk_client.go]: can you rename all variables named sb to wr?

File: api.go (hunk 1 above)
Comment (hananothman) [reply api.go]: Done, thanks!

File: api.go (hunk 2 above)
Comment (hananothman) [reply api.go]: Yes, and it would pass that through as the output.

File: webrisk_client.go (hunk 3 above)
Comment (hananothman) [reply webrisk_client.go]: Done.

**Comment by hananothman** [comment]
Thanks JJ! PTAL (the branch required so much pulling for some reason, sorry about those commits)

File: api.go (hunk 2 above)
Comment (JJLionHeart) [reply api.go]: Can you add a test case for this? just to be sure

Comment (hananothman) [reply api.go]: Added.

**Pull request merged**